src/components/timebank/specific-time-entries-card.tsx

```diff
@@ -63,6 +63,8 @@ const SpecificTimeEntriesCard = ({ selectedEmployeeId }: Props) => {
     end: todayOrEarlier
   });
 
+  const [open, setOpen] = useState(false);
```

Comment:
This seems like a fine solution, can you also implement the same solution for the date pickers if the user clicks the by range checkbox